src/index.js

```diff
@@ -0,0 +1,80 @@
+import './style.scss';
+import TaskList from './modules/taskList.js';
+
+const { body } = document;
+// Container Section
+const conSec = document.createElement('section');
+conSec.classList.add('section');
+
+// Top Section
+const topSec = document.createElement('div');
+topSec.classList.add('topSec');
+const title = document.createElement('h1');
+title.textContent = 'Today\'s To Do';
+const reloadBtn = document.createElement('button');
+reloadBtn.innerHTML = '<i class="fa-solid fa-arrows-rotate"></i>';
+
+// Window Reload
+reloadBtn.addEventListener('click', () => {
+  const taskList = new TaskList();
+  taskList.clearAll();
+});
+
+// Bottom Section
+const botSec = document.createElement('div');
+botSec.classList.add('botSec');
+const title2 = document.createElement('h2');
+title2.textContent = 'Clear all completed';
+
+// Clear All Event
+title2.addEventListener('click', () => {
+  const taskList = new TaskList();
+  taskList.clearAllCompleted();
+});
```

Comment:
The 'Clear All Button' should only delete the tasks marked as completed instead of everything. One way you could fix this is by using the same taskList that contains your tasks instead of creating a new one with `new TaskList()`, otherwise your list of tasks will be replaced with the new *(empty)* list.